### PR TITLE
feat: implement loan tags on category pages

### PR DESCRIPTION
**Pull request by egoroza**
Screen Shot 2022-10-31 at 7 33 58 PM
Screen Shot 2022-10-31 at 7 34 09 PM
Screen Shot 2022-10-31 at 7 34 37 PM
Screen Shot 2022-10-31 at 7 34 41 PM
Screen Shot 2022-10-31 at 7 34 48 PM
Screen Shot 2022-10-31 at 7 34 54 PM

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
 		"@graphql-tools/url-loader": "^7.12.1",
 		"@kiva/kv-components": "^3.7.1",
 		"@kiva/kv-tokens": "^2.4.0",
-		"@mdi/js": "^5.9.55",
+		"@mdi/js": "^7",
```
Comment (mcstover) [inline package.json:53]: What did this add? We'll need to update in cms-page-server as well...

**Review by mcstover — APPROVED** [review]
This looks nice!

File: package.json (hunk 1 above)
Comment (egoroza) [reply package.json:53]: new icons!

Comment (mcstover) [reply package.json:53]: :)

**Review by dyersituations — APPROVED** [review]

**Pull request merged**